src/AppBundle/DataFixtures/ORM/LoadNoticeData.php

```diff
@@ -12,8 +12,13 @@ class LoadNoticeData extends AbstractFixture implements DependentFixtureInterfac
 {
     public function load(ObjectManager $manager)
     {
+        $johnDoe = $this->getReference('john_doe');
+        $janeDoe = $this->getReference('jane_doe');
+        $famousContributor = $this->getReference('famous_contributor');
+        $contributor2 = $this->getReference('contributor2');
+
         $notice = new Notice();
-        $notice->setContributor($this->getReference('contributor'));
+        $notice->setContributor($johnDoe);
```

Comment:
Did you want to add relayer here and there since you edited this file ?
Just saw you added the `addRelayer` sugar to abstract a bit the `relay` relation. :+1:

Reply:
I did, actually, in line 92 and 93 of the same file